tag_hierarchy/commands/nodes.cpp

```diff
@@ -60,7 +60,8 @@ Nodes::ProcessRequest(std::vector<NodeType> &nodes)
 
     auto valid_nodes = std::set<VertexT>();
     auto valid_models = std::map<VertexT, std::set<VertexT>>();
-    auto dfs_visitor = FilteredHierarchyVisitor(valid_nodes, valid_models, kpifilter);
+    auto suppressedNodes = std::set<VertexT>();
```

Comment:
It would be nice to keep the snake_case consistent for local variables

Reply:
Done